make/aarch64_check_codec_order.py

```diff
@@ -0,0 +1,88 @@
+# **********************************************************
```

Comment:
Perhaps a `#!` line would be useful, in case someone's running it from the command line. But I'm not sure what it should be:
```
$ head -n 1 `find -name '*.py'`
==> ./make/upload.py <==
#!/usr/bin/env python

==> ./core/arch/aarch64/codec.py <==
#!/usr/bin/python

==> ./tools/libdynamorio.so-gdb.py <==
#!/usr/bin/env python
```

Reply:
I think `#!/usr/bin/env python` has the advantage that it will pick the right Python interpreter if you are in a virtualenv. I'll add it (and also make the file executable)